lib/omniauth/strategies/openid_connect.rb

```diff
@@ -111,7 +111,7 @@ def callback_phase
         invalid_state = params['state'].to_s.empty? || params['state'] != stored_state
 
         raise CallbackError.new(params['error'], error_description, params['error_uri']) if error
-        raise CallbackError, 'Invalid state parameter' if invalid_state
+        raise CallbackError.new(:csrf_detected, "Invalid state parameter") if invalid_state
```

Comment:
Please, revert this change back to old version

https://github.com/rubocop-hq/ruby-style-guide#exception-class-messages

Reply:
in most cases it is right to follow the guide, but i think in that case it isn't.
without raising the error instance it is not possible to provide more informations to the error.